DataFormats/Reconstruction/include/ReconstructionDataFormats/MatchInfoTOF.h

```diff
@@ -25,20 +25,21 @@ namespace dataformats
 {
 class MatchInfoTOF
 {
-  using evGIdx = o2::dataformats::EvIndex<int, o2::dataformats::GlobalTrackID>;
+  //  using evGIdx = o2::dataformats::EvIndex<int, o2::dataformats::GlobalTrackID>;
+  using evGIdx = o2::dataformats::GlobalTrackID;
   using evIdx = o2::dataformats::EvIndex<int, int>;
 
  public:
-  MatchInfoTOF(evIdx evIdxTOFCl, double time, float chi2, o2::track::TrackLTIntegral trkIntLT, evGIdx evIdxTrack, float dt = 0, float z = 0) : mEvIdxTOFCl(evIdxTOFCl), mSignal(time), mChi2(chi2), mIntLT(trkIntLT), mEvIdxTrack(evIdxTrack), mDeltaT(dt), mZatTOF(z){};
+  MatchInfoTOF(int idLocal, evIdx evIdxTOFCl, double time, float chi2, o2::track::TrackLTIntegral trkIntLT, evGIdx evIdxTrack, float dt = 0, float z = 0) : mIdLocal(idLocal), mEvIdxTOFCl(evIdxTOFCl), mSignal(time), mChi2(chi2), mIntLT(trkIntLT), mEvIdxTrack(evIdxTrack), mDeltaT(dt), mZatTOF(z){};
   MatchInfoTOF() = default;
   void setEvIdxTOFCl(evIdx index) { mEvIdxTOFCl = index; }
   void setEvIdxTrack(evGIdx index) { mEvIdxTrack = index; }
   evIdx getEvIdxTOFCl() const { return mEvIdxTOFCl; }
   evGIdx getEvIdxTrack() const { return mEvIdxTrack; }
   int getEventTOFClIndex() const { return mEvIdxTOFCl.getEvent(); }
   int getTOFClIndex() const { return mEvIdxTOFCl.getIndex(); }
-  int getEventTrackIndex() const { return mEvIdxTrack.getEvent(); }
-  int getTrackIndex() const { return mEvIdxTrack.getIndex(); }
+  evGIdx getEventTrackIndex() const { return mEvIdxTrack; }
```

Comment:
We don't need anymore getEventTrackIndex.

Reply:
Done

Reply:
Then, let's remove misleading ..EvIdx..., i.e. the methods should be 
````
void setTOFClRef(int index) { mTOFClRef = index; }
void setEvIdxTrack(GTrackID gid) { mTrackRef = gid; }
int getTOFClRef() const { return mTOFClRef; }
GTrackID getTrackRef() const { return mTrackRef; }
GTrackID getTrackIndex() const { return mTrackRef; } //  in order to not break the QC
````

Reply:
actually, please change `GTrackID getTrackIndex() const { return mTrackRef; }` to `int getTrackIndex() const { return mTrackRef.getIndex(); }`, just to not break the QC.

@njacazio eventually the QC should use at
https://github.com/AliceO2Group/QualityControl/blob/3bd3818639a8d9e95393a30b4b1f4d33e00d448a/Modules/TOF/src/TOFMatchedTracks.cxx#L123
https://github.com/AliceO2Group/QualityControl/blob/3bd3818639a8d9e95393a30b4b1f4d33e00d448a/Modules/TOF/src/TOFMatchedTracks.cxx#L137
https://github.com/AliceO2Group/QualityControl/blob/3bd3818639a8d9e95393a30b4b1f4d33e00d448a/Modules/TOF/src/TOFMatchedTracks.cxx#L159
`matchTOF.getTrackRef().getIndex()` instead of `matchTOF.getTrackIndex()`

Reply:
Done

Reply:
Thanks a lot @shahor02! We will handle it